src/TimeGridEvent.js

```diff
@@ -40,29 +41,44 @@ function TimeGridEvent(props) {
     </div>,
   ]
 
+  const eventStyle = isBackgroundEvent
+    ? {
+        ...userProps.style,
+        top: stringifyPercent(top),
+        height: stringifyPercent(height),
+        width: `calc(${width}% + 10px)`,
```

Comment:
Is the additional 10px here to compensate for the right margin on events?  It might be worth adding a comment to explain why we need this.

Reply:
Good point! Also, there was a small inconsistency causing these events to not render at 100% width of the column. I'll add a comment and fix this behavior.

Also, I'll update the examples screenshots.